packages/xrpl/snippets/src/bridgeTransfer.ts

```diff
@@ -0,0 +1,166 @@
+/* eslint-disable @typescript-eslint/consistent-type-assertions -- needed here */
+/* eslint-disable no-await-in-loop -- needed here */
+import {
+  AccountObjectsRequest,
+  LedgerEntry,
+  Client,
+  XChainAccountCreateCommit,
+  XChainBridge,
+  XChainCommit,
+  XChainCreateClaimID,
+  xrpToDrops,
+  Wallet,
+  TransactionMetadata,
+} from '../../src'
+import { CreatedNode } from '../../src/models/transactions/metadata'
+
+async function sleep(sec: number): Promise<void> {
+  return new Promise((resolve) => {
+    setTimeout(resolve, sec * 1000)
+  })
+}
+
+const lockingClient = new Client(
+  'wss://sidechain-net1.devnet.rippletest.net:51233',
+)
+const issuingClient = new Client(
+  'wss://sidechain-net2.devnet.rippletest.net:51233',
+)
+const MAX_LEDGERS_WAITED = 5
+const LEDGER_CLOSE_TIME = 4
+
+void bridgeTransfer()
+
+async function bridgeTransfer(): Promise<void> {
+  await lockingClient.connect()
+  await issuingClient.connect()
+  const lockingChainDoor = 'rMAXACCrp3Y8PpswXcg3bKggHX76V3F8M4'
+
+  const accountObjectsRequest: AccountObjectsRequest = {
+    command: 'account_objects',
+    account: lockingChainDoor,
+    type: 'bridge',
+  }
+  const bridgeData = (await lockingClient.request(accountObjectsRequest)).result
+    .account_objects[0] as LedgerEntry.Bridge
```

Comment:
Is this always guaranteed to be the 1st result? (If not, maybe it'd be better for the snippet to show how to consistently find the bridge entry? Or add a little helper that the snippet can use for that?)

Reply:
(And if it is guaranteed, I would add a comment above saying so to help future developers know that's the recommended way to look up this field)

Reply:
In this particular case (the example) this bridge only has one bridge, so yes it's guaranteed. But door accounts can have multiple bridges.

Reply:
In that case, I'd recommend being able to find the specific bridge if possible (or if having multiple bridges is a severe edge case then maybe just leaving a comment about it)